src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -258,6 +275,36 @@ private function findHeaderCommentInsertionIndex(Tokens $tokens)
         return $next + 1;
     }
 
+    /**
+     * Find the index where the header comment must be inserted if location is after_namespace.
+     *
+     * @param Tokens $tokens
+     *
+     * @return int
+     */
+    private function findHeaderCommentInsertionIndexForNamespace(Tokens $tokens)
+    {
+        $index = $tokens->getNextMeaningfulToken(0);
+        while (null !== $index && !$tokens[$index]->isGivenKind(T_NAMESPACE)) {
+            $index = $tokens->getNextMeaningfulToken($index);
+        }
+
+        if (null === $index) {
+            return $this->findHeaderCommentInsertionIndexForDeclareStrict($tokens);
+        }
+
+        $next = $tokens->getNextMeaningfulToken($index);
+        while (null !== $next && !$tokens[$next]->equals(';')) {
+            $next = $tokens->getNextMeaningfulToken($next);
+        }
```

Comment:
No, it would insert the header above the namespace. I don't think there is a good solution where to insert the header. As the declaration would have to insert it *after* the namespace (thus after closing curly bracket) but that IMHO would not make any sense. I guess the expectation would be to insert the header *inside* but that's not covered by the configuration.